include/lsst/afw/coord/Coord.h

```diff
@@ -131,7 +131,7 @@ class Coord {
     void rotate(Coord const &axis, lsst::afw::geom::Angle const theta);
     lsst::afw::geom::Angle offset(lsst::afw::geom::Angle const phi, lsst::afw::geom::Angle const arcLen);
     
-    Coord::Ptr convert(CoordSystem system) const;
+    Coord::Ptr convert(CoordSystem system, double epoch=2000) const;
```

Comment:
Since you're touching this line, might as well switch from `Coord::Ptr` to `PTR(Coord)` at the same time.